Release 4.1.0 - Build: "failed to execute goal maven-surefire-plugin:test" (openjdk 11, maven 3.6)

same as  #292 (not backported yet)
related to #291 jacoco java 11 support

> ## Observed behaviour
> error: `Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M1:test`
> ```
> [INFO] asqatasun-api 4.1.0-SNAPSHOT ....................... SUCCESS [  1.873 s]
> [INFO] reference 4.1.0-SNAPSHOT ........................... SUCCESS [  0.627 s]
> [INFO] parameterization 4.1.0-SNAPSHOT .................... FAILURE [  1.031 s]
> (...)
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M1:test (default-test) on project asqatasun-parameterization: There are test failures.
> [ERROR] 
> [ERROR] Please refer to engine/parameterization/target/surefire-reports for the individual test results.
> [ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
> [ERROR] The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
> [ERROR] Command was /bin/sh -c cd engine/parameterization && /usr/lib/jvm/java-11-openjdk-amd64/bin/java -javaagent:/.m2/repository/org/jacoco/org.jacoco.agent/0.8.1/org.jacoco.agent-0.8.1-runtime.jar=destfile=engine/parameterization/target/jacoco.exec -jar engine/parameterization/target/surefire/surefirebooter15039007492587206080.jar engine/parameterization/target/surefire 2019-07-16T04-48-05_845-jvmRun1 surefire1160939195451314008tmp surefire_016365646853849541748tmp
> ```
> 
> 
> ## Steps to reproduce the behaviour
> ```
> cd ~
> mv .m2 .m2_backup
> git clone ...
> cd Asqatasun
> mvn clean install
> ```
> related to #291 jacoco java 11 support
> 
> ##  Configuration
> 
> ```
> mvn --version
> 	Apache Maven 3.6.0
> 	Maven home: /usr/share/maven
> 	Java version: 11.0.3, vendor: Oracle Corporation, runtime: /usr/lib/jvm/java-11-openjdk-amd64
> 	OS name: "linux", version: "4.15.0-54-generic", arch: "amd64", family: "unix"
> 
> java -version
> 	openjdk version "11.0.3" 2019-04-16
> ```
>

--- a/pom.xml
+++ b/pom.xml
@@ -129,7 +129,7 @@
         <build-helper-maven-plugin.version>3.0.0</build-helper-maven-plugin.version>
         <buildnumber-maven-plugin.version>1.4</buildnumber-maven-plugin.version>
         <dependency-check-maven.version>3.1.2</dependency-check-maven.version>
-        <jacoco-maven-plugin.version>0.8.1</jacoco-maven-plugin.version>
+        <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
         <markdown-page-generator-plugin.version>2.1.0</markdown-page-generator-plugin.version>
         <yuicompressor-maven-plugin.version>1.5.1</yuicompressor-maven-plugin.version>
     </properties>

